todolist/index.html

```diff
@@ -18,6 +18,22 @@
     <div id="text-area">
       <h2>Today</h2>
     </div>
+    <!-- deactivated -->
+    <div id="my-modal" class="modal modal-deactivated"> 
+      <div class="modal-content">
+        <span class="cancle-button">
+          <a class="cancle-button">&times;</a>
+        </span>
+        <div class="modal-content-name">
+          <input type="checkbox" name="modal-content-checkbox" id="modal-content-checkbox">
+          Name
+        </div>
+        <div class="modal-content-details">
+          <span class="modal-content-details-date">Date : 2021-07-10</span><br><br>
+          <span class="modal-content-details-deleteBtn">Delete</span>
+        </div>
```

Comment:
```suggestion
        <div class="modal-content-details">
          <span class="date">Date : 2021-07-10</span><br><br>
          <button  type="button" class="delete-btn">Delete</button>
        </div>
```

- 클래스 이름에 대문자는 사용하지 않습니다.
- 부모에서 이미 클래스 이름에 `modal-content-details` 정보가 있는데 자식들에게 굳이 이 정보를 추가적으로 붙일 필요는 없다고 봅니다.
- 버튼같이 클릭 가능한 요소에는 `button` 태그를 사용해야 합니다. 그리고 스타일을 CSS로 커스텀해서 주는 것이 좋습니다.